README.md

```diff
@@ -220,11 +220,16 @@ or by defining the subsiteCMSShowInMenu function in your admin:
 
 ### Using Subsites in combination with Fluent
 
-When using Subsites in combination with Fluent module, the Subsites module sets the i18n locale to the language defined in the current Subsite. When this behaviour is not desired and you need to use the locale in FluentState use the following setting in your yml config file:
+When using Subsites in combination with Fluent module, the Subsites module sets the i18n locale to the language defined in the current Subsite. When this behaviour is not desired and you need to use the locale in FluentState you can set the current Fluent locale in your PageController like this:
 
-```yaml
-    SilverStripe\Subsites\Extensions\SiteTreeSubsites:
-      ignore_subsite_locale: true
+*PageController.php*
+```
+:::php
+     protected function init(){
+        if ($this->dataRecord->SubsiteID !== 0){
+            i18n::set_locale(FluentState::singleton()->getLocale());
+        }
+     }
 ```
```

Comment:
Can you please explain why you're making this change? What was wrong with the yaml configuration documented here?

Reply:
I make this change, because the yaml configuration documented is not in use in the Subsite-Module. So even with the yml configuration set, the i18n-Locale is still the one defined for the subsite and not the one from fluent. My solution is just one possible workaround.

Reply:
> I make this change, because the yaml configuration documented is not in use in the Subsite-Module. 

That yaml configuration is used here:
https://github.com/silverstripe/silverstripe-subsites/blob/4e292b8e89f9ee214340a215285cdcb526a2b5f8/src/Extensions/SiteTreeSubsites.php#L399-L417

Reply:
ah I see - okay. In module version 2.6.0 it wasn't there :). Seems it changed in the meanwhile. 
Than my change is unnecessary of course .

Reply:
Okay cool. I'll close this PR then. Thank you for your efforts and patience though, it's definitely appreciated.